static/app/components/searchQueryBuilder/filterKeyOperator.tsx

```diff
@@ -206,6 +209,15 @@ export function FilterKeyOperator({token, state, item}: FilterOperatorProps) {
       options={options}
       value={operator}
       onChange={option => {
+        trackAnalytics('search.operator_autocompleted', {
+          organization,
+          query,
+          search_type: savedSearchType === 0 ? 'issues' : 'events',
```

Comment:
Thoughts on a function to convert the enum to a display string? If other product areas adopt this `savedSearchType` will probably end up not being 0 or 1 and it seems probable we'll miss updating at least one of these

Reply:
You make a good point - I pulled this logic over from SmartSearchBar without really thinking about it, but this isn't the right way to be logging this with all the new use cases. I'll follow up with that function